packages/webapp/components/search/SearchControlPage.tsx

```diff
@@ -31,6 +41,18 @@ const Search = (): ReactElement => {
   return (
     <>
       <NextSeo {...seo} {...baseSeo} />
+      {isSearchV1 && (
+        <SearchProviderButton
+          className="mt-4"
+          provider={SearchProviderEnum.Chat}
+          query={query.q as string}
+        >
+          <span className="text-white">Search posts on daily.dev instead</span>
```

Comment:
It's `Search posts` right? (from design what I can see)

Reply:
Yeah, here its actually the query + "Ask daily.dev AI", updated to show query in 2adc2c4df05a00d28675a78a69ef7d66c37d0f7b